osu.Game/Rulesets/Edit/HitObjectComposer.cs

```diff
@@ -178,7 +178,7 @@ protected override void UpdateAfterChildren()
         /// <summary>
         /// Creates a <see cref="ScalableContainer"/> which provides a layer above or below the <see cref="Playfield"/>.
         /// </summary>
-        protected virtual Container CreateLayerContainer() => new Container { RelativeSizeAxes = Axes.Both };
+        protected virtual Container CreateLayerContainer() => DrawableRuleset.CreatePlayfieldAdjustmentContainer();
```

Comment:
Does this still need to exist?

Reply:
As in the virtual part? See the osu! one.

Reply:
I mean you removed it in the osu! one, really this shouldn't be required since it's handled by the DrawableRuleset now.